assets/javascripts/discourse/components/fields/da-categories-field.gjs

```diff
@@ -0,0 +1,38 @@
+import Category from "discourse/models/category";
+import { action } from "@ember/object";
+import BaseField from "./da-base-field";
+import DAFieldLabel from "./da-field-label";
+import DAFieldDescription from "./da-field-description";
+import CategorySelector from "select-kit/components/category-selector";
+import { fn, hash } from "@ember/helper";
+
+export default class CategoriesField extends BaseField {
+  <template>
+    {{! template-lint-disable no-redundant-fn }}
+    <section class="field categories-field">
+      <div class="control-group">
+        <DAFieldLabel @label={{@label}} @field={{@field}} />
+
+        <div class="controls">
+          <CategorySelector
+            @categories={{this.categories}}
+            @onChange={{fn this.onChangeCategories}}
```

Comment:
There might be a better way to do this, but if I don't use `fn`, I will get a Cell which is not working with all select-kit code still checking on `this.attrs.onChange`